src/odemis/driver/piezomotor.py

```diff
@@ -180,6 +180,9 @@ def __init__(self, name, role, port, axes, inverted=None, param_file=None, **kwa
         self._port = None  # port number
         min_axis = min(self._axis_map.values())
         self._serial = self._findDevice(port, min_axis)
+        # To be able to give the correct response the simulator needs to know which axes have closed loop active.
+        if port.startswith("/dev/fake"):
+            self._serial.closed_loop = {self._axis_map[k]: v for (k, v) in self._closed_loop.items()}
```

Comment:
No. The real hardware doesn't need this info, so there is no reason the simulator needs it. Please, make the simulator behave like the real hardware: based on the commands it receives it switches between closed-loop and open-loop. I think for the simulator it should be enough to have this behaviour (not sure if the real hardware has extra rules):
 * runAbsTargetMove() and runRelTargetMove() set closed_loop to True.
 * stopAxis sets the closed_loop to False